File: 프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    dic = {0 : 'THU', 1: 'FRI', 2: 'SAT', 3: 'SUN', 4: 'MON', 5: 'TUE', 6: 'WED'}
    month_days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    
    # 일수 계산
    days = sum(month_days[:a - 1]) + b
    
    return dic[days % 7]

File: 프로그래머스/1/12901. 2016년/2016년.js
function solution(a, b) {
    var answer = '';
    let week = ['THU', 'FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED'];
    let month_days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
    
    let days = b;
    
    for (let i = 0; i < a - 1; i++) {
        days += month_days[i];
    }
    
    return week[days%7];
}